### PR TITLE
Update README to reflect latest project status

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Re-work the readme which still referenced this as a "coming soon" type project. Now I've established core feature set, linked to the online docs, and included a couple of screenshots (can be improved later) to make the package seem more appealing and "ready"

To view the README in a formatted mode you can go to: https://github.com/FlowFuse/node-red-dashboard/blob/readme-updates/README.md

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,35 +24,52 @@ If you want to use `npm` to install your nodes, you can instead [follow these in
...
+
+<img width="600" alt="Example of data visualisations in Dashboard 2.0" src="https://github.com/FlowFuse/node-red-dashboard/assets/99246719/48da3687-bebd-4e2a-88b5-6abe2af4fc46">
+
+No dashboard is complete without data visualisation. Node-RED Dashboard 2.0 provides a core `ui-chart` widget to provide a simple, yet powerful way to visualise your data. It supports a wide range of chart types, including line, bar and scatter, with more planned for the near future.
+
+### Flexible Customisation
+
+<img width="600" alt="Example of a dashboard using custom templates to render a to-do list" src="https://github.com/FlowFuse/node-red-dashboard/assets/99246719/10d2b81b-8dfe-4597-84c8-03c1776287d5">
+
+As with Dashboard 1.0, the new Dashboard 2.0, comes with a `ui-template` node which allows you define your own custom widgets and styling.
```
Comment (MarianRaphael) [inline README.md:45]:
```suggestion
As with Dashboard 1.0, the new Dashboard 2.0, comes with a `ui-template` node which allows you to define your own custom widgets and styling. 
```

File: README.md [hunk 2]
```diff
@@ -18,35 +24,52 @@ If you want to use `npm` to install your nodes, you can instead [follow these in
...
+The new `ui-markdown` widget allows you to build rich, interactive applications using [Markdown](https://www.markdownguide.org/) and [Mermaid Charts](https://mermaid.js.org/). You can use Markdown templates to build your UI, and populate it with your own, dynamic data, and then use Mermaid to display dynamic charts and diagrams.
 
-## Community and Contribution
+## Project Planning
 
-We strongly believe in the power of community. If you have suggestions, feedback, or features you'd like to see, please open a [Github issue](https://github.com/FlowFuse/node-red-dashboard/issues/new/choose). We also highly encourage open-source contributions. We aim to create a robust tool through community collaboration.
+### Community and Contribution
 
-## Join Us
+As an open-source project, Node-RED Dashboard 2.0 openly welcomes all forms of contributions, whether that's ideas, bug reports, or code contributions through Pull Requests.
```
[the remark this answers is not in the document]
Comment (joepavitt) [reply README.md:63]: that's === that is

[the remark this answers is not in the document]
Comment (joepavitt) [reply README.md:63]: which is appropriate here

[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply README.md:63]: `whether those are` would've been correct too?

[the remark this answers is not in the document]
Comment (MarianRaphael) [reply README.md:63]: I thought that 'ideas, bug reports, or code contributions' being plural meant it should be 'that are,' but okay. Besides this, it's all right.

**Review by MarianRaphael — APPROVED** [review]

**Pull request merged**